### PR TITLE
Client redesign

### PR DESCRIPTION
**Pull request by MaskaSaw**

### CODE REVIEW

File: hotels-client-frontend/src/app/authentication/authentication.component.less [hunk 1]
```diff
@@ -1,6 +1,7 @@
 input.form-control {
-  	width: 20%;
-	margin-left: 40%;
+  	width: 40%;
+	margin-left: 30%;
+	min-width: 100px;
```
Comment (andrei-kruhliakou-itechart) [inline hotels-client-frontend/src/app/authentication/authentication.component.less]: Indents

File: hotels-client-frontend/src/app/authentication/authentication.component.less [hunk 2]
```diff
@@ -26,4 +27,18 @@ input.form-control {
 		box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
 	}
 
+}
+
+@media screen and (min-width: 700px) {
+    input.form-control {
+		width: 30%;
+	  	margin-left: 35%;
```
Comment (andrei-kruhliakou-itechart) [inline hotels-client-frontend/src/app/authentication/authentication.component.less]: Indents

File: hotels-client-frontend/src/app/authentication/authentication.component.less [hunk 3]
```diff
@@ -26,4 +27,18 @@ input.form-control {
...
+  }
+}
+
+@media screen and (min-width: 1200px) {
+    input.form-control {
+		width: 16%;
+	  	margin-left: 42%;
```
Comment (andrei-kruhliakou-itechart) [inline hotels-client-frontend/src/app/authentication/authentication.component.less]: Indents

File: hotels-client-frontend/src/app/hotels/hotels.component.less [hunk 4]
```diff
@@ -62,9 +62,15 @@ example-form {
   width: 100%;
 }
 
-.example-full-width {
-  width: 15%;
-  margin: 0 10px 0 10px;
+.datepicker-form-control {
+  display: block !important;
```
Comment (andrei-kruhliakou-itechart) [inline hotels-client-frontend/src/app/hotels/hotels.component.less:66]: Avoid using !implortant unless absolutely necessary

**Review by andrei-kruhliakou-itechart — CHANGES_REQUESTED** [review]

File: hotels-client-frontend/src/app/hotels/hotels.component.less (hunk 4 above)
Comment (MaskaSaw) [reply hotels-client-frontend/src/app/hotels/hotels.component.less:66]: Yeah, could not find another solution and used it like a last chance here

Comment (andrei-kruhliakou-itechart) [reply hotels-client-frontend/src/app/hotels/hotels.component.less:66]: And what exactly is the problem here?

Comment (MaskaSaw) [reply hotels-client-frontend/src/app/hotels/hotels.component.less:66]: Inline styles of datepicker are always used before any class parameters from my .less files, and i need to change it's display style to block for right positioning on low-width screens

**Review by andrei-kruhliakou-itechart — APPROVED** [review]

**Pull request merged**